VAMobile/src/components/CommonErrorComponents/ErrorAlert.tsx

```diff
@@ -33,11 +33,11 @@ const ErrorAlert: FC<ErrorAlertProps> = ({ onTryAgain, title, text }) => {
   return (
     <VAScrollView contentContainerStyle={scrollStyles}>
       <Box justifyContent="center" {...containerStyles}>
-        <AlertBox border="error" title={title || t('errorAlert.title')} text={text}>
+        <AlertWithHaptics variant="error" header={title || t('errorAlert.title')} description={text}>
           <Box mt={theme.dimensions.standardMarginBetween} accessibilityRole="button">
             <Button onPress={onTryAgain} label={t('refresh')} testID={t('refresh')} />
```

Comment:
Could potentially be simplified moving this to the `primaryButton` prop to AlertWithHaptics.